src/NuGet.Core/NuGet.Protocol/Resources/PackageMetadataResourceV3.cs

```diff
@@ -38,37 +55,240 @@ public class PackageMetadataResourceV3 : PackageMetadataResource
             Common.ILogger log,
             CancellationToken token)
         {
-            var metadataCache = new MetadataReferenceCache();
-
-            var packages =
-                (await _regResource.GetPackageMetadata(packageId, includePrerelease, includeUnlisted, sourceCacheContext, log, token))
-                    .Select(ParseMetadata)
-                    .Select(m => metadataCache.GetObject(m))
-                    .ToArray();
+            var range = VersionRange.All; // This value preset for Consolidate UI.
```

Comment:
Are you sure this method is specific to Consolidate? I think several UI codepaths come here.

Reply:
I changed wording. Actually it's from "Update UI" too.